cordova-react/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",         
+        "jsx": "react",
+        "sourceMap": true,
+        "noImplicitAny": true
+    },
+    "files": [
```

Comment:
i do not know. :) does debugging work for you on all platforms with these configurations?